File: test_1/main.py
import sys

# alpha = input() 
alpha = sys.argv[1]
cnt = 0
if alpha < 'A' or alpha > 'Z':
  print("standard error", file=sys.stderr)
  exit(1)

for i in range(ord(alpha),ord('Z')+1):
  cnt+=1
  print(chr(i),end="")

print()
print(cnt)

File: test_1/main.php
<?php

// $word = fgets(STDIN);
$word = $argv[1];
$count = 0;
if ($word < "A" || $word > "Z") {
    $msg_error = 'standard error' . PHP_EOL;
    fputs(STDERR, $msg_error);
    exit();
}

for ($i = ord($word); $i <= ord("Z"); $i++) {
    $count++;
    echo chr($i);
}

echo PHP_EOL;
echo $count;
?>
